deployment/deploy_ingestion.sh

```diff
@@ -0,0 +1,112 @@
+
+    RESOURCES_JSON=$(cat resources.json)
+    jq --arg policy "${CLOUDWATCH_POLICY}" '.cloudwatch_log_policy |= $policy' <<< $RESOURCES_JSON > resources.json
+else
+    echo "Cloudwatch policy already created, skipping..."
+fi
+wait
+
+if [ "$S3_READ_WRITE_POLICY" == "" ] || [ "$S3_READ_WRITE_POLICY" == null ]; then
+    echo "Creating s3 read/write policy from template..."
+    
+    S3_READ_WRITE_POLICY=$(aws iam create-policy --policy-name s3-ingestion-${SUFFIX} --policy-document "${S3_READ_WRITE_JSON}" | jq .Policy.Arn | tr -d '"')
+
+    RESOURCES_JSON=$(cat resources.json)
+    jq --arg policy "${S3_READ_WRITE_POLICY}" '.s3_read_write_policy |= $policy' <<< $RESOURCES_JSON > resources.json
+else
+    echo "S3 policy already created, skipping..."
+fi
+wait 
+
+if [ "$IAM_ROLE" == "" ] || [ "$IAM_ROLE" == null ]; then
+    echo "Creating IAM role to carry policies..."
+
+    IAM_ROLE=$(aws iam create-role --role-name lambda-role-${SUFFIX} \
+    --assume-role-policy-document file://templates/trust.json | jq .Role.Arn | tr -d '"')
+
+    RESOURCES_JSON=$(cat resources.json)
+    jq --arg policy "${IAM_ROLE}" '.iam_role |= $policy' <<< $RESOURCES_JSON > resources.json
+else
+    echo "IAM role already created, skipping..."
+fi
+wait
+
+echo 'Attaching policies to IAM role...'
+aws iam attach-role-policy --policy-arn ${CLOUDWATCH_POLICY} --role-name lambda-role-${SUFFIX}
+aws iam attach-role-policy --policy-arn ${S3_READ_WRITE_POLICY} --role-name lambda-role-${SUFFIX}
+wait
+
+echo 'Creating lambda function...'
+FUNCTION=$(aws lambda create-function --function-name ${FUNCTION_NAME} --runtime python3.9 --role ${IAM_ROLE} --package-type Zip --handler test-ingestion.ingestion_handler --code S3Bucket=${CODE_BUCKET_NAME},S3Key=${FUNCTION_NAME}/test-ingestion.zip)
+wait
+
+if [ "$EVENTBRIDGE_RULE" == "" ] || [ "$EVENTBRIDGE_RULE" == null ]; then
+    echo "Creating eventbridge rule..."
+
+    EVENTBRIDGE_RULE=$(aws events put-rule --name ingestion-rule-${SUFFIX} --schedule-expression "rate(5 minutes)" | jq .RuleArn | tr -d '"')
+
+    RESOURCES_JSON=$(cat resources.json)
+    jq --arg policy "${EVENTBRIDGE_RULE}" '.eventbridge_rule |= $policy' <<< $RESOURCES_JSON > resources.json
+else
+    echo "Eventbridge rule already created, skipping..."
+fi
+wait
+
+echo "Giving permission to eventbridge rule to invoke lambda function..."
+aws lambda add-permission --function-name ${FUNCTION_NAME} --principal events.amazonaws.com --statement-id eventbridge-invoke-${SUFFIX} --action "lambda:InvokeFunction" --source-arn ${EVENTBRIDGE_RULE} 
+wait
+
+TARGET_JSON=$(jq --arg aws_id "${AWS_ACCOUNT_ID}" --arg func_name "${FUNCTION_NAME}" '.Arn |= "arn:aws:lambda:us-east-1:" + $aws_id + ":function:" + $func_name' templates/eventbridge_targets.json)
+echo "Adding lambda function as target for eventbridge rule..."
+aws events put-targets --rule ingestion-rule-${SUFFIX} --targets "[${TARGET_JSON}]"
```

Comment:
This command gives output to the console, requires user to manually quit in order to continue. Recommend outputting output to /dev/null instead.
<img width="414" alt="Screenshot 2022-12-16 at 11 54 12" src="https://user-images.githubusercontent.com/59575217/208093006-7dd4f1f2-0cd2-448b-9949-21faa1e6b06d.png">